bundles/alpha.model/src/alpha/model/ComplexityCalculator.xtend

```diff
@@ -24,11 +24,6 @@ class ComplexityCalculator extends AbstractAlphaCompleteVisitor {
 	int maxComplexity
 
 	def static int complexity(AlphaVisitable av) {
-		
-		val system = av.getContainerSystem
-		if (system !== null && system.parameterDomain.dim(ISLDimType.isl_dim_param) > 1)
-			throw new UnsupportedOperationException("Current complexity calculator only works for systems containing a single parameter.")
-		
```

Comment:
This isn’t strictly needed for the complexity calculator as it’s used by simplification. Now if there are multiple parameters, e.g. [N,W], a complexity of 3 could either mean (N^2)W or NW^2.